### PR TITLE
Enable OCP3 builds and add C2S rules

### PR DESCRIPTION
**Pull request by redhatrises**
This starts to add OCP rules and content. Also uses a top-level `applications` directory for applications that could/can be installed for multiple different OSes.

### CODE REVIEW

File: applications/benchmark.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    content structured in the eXtensible Configuration Checklist Description Format (XCCDF)
+    in order to support security automation.  The SCAP content is
+    is available in the <tt>scap-security-guide</tt> package which is developed at
+    {{{ weblink(link="https://www.open-scap.org/security-policies/scap-security-guide") }}}.
+    <br/><br/>
+    Providing system administrators with such guidance informs them how to securely
+    configure systems under their control in a variety of network roles. Policy
+    makers and baseline creators can use this catalog of settings, with its
+    associated references to higher-level security control catalogs, in order to
+    assist them in security baseline creation. This guide is a <em>catalog, not a
```
Comment (shawndwells) [inline applications/benchmark.yml]: Could we ``<em>`` the entire sentence -- e.g. ``<em>This guide is a....</em>``?

File: applications/openshift/ocp-permissions/ocp-files/directory_permissions_etc_origin/rule.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+documentation_complete: true
+
+title: 'The OpenShift Configuration Directory Must Have Mode 0700'
```
[the remark this answers is not in the document]
Comment (redhatrises) [reply applications/openshift/ocp-permissions/ocp-files/directory_permissions_etc_origin/rule.yml]: This is content is not all finalized, so a separate ticket should be opened for finalization fixes. Also, not using `c2s` creates confusing between the RHEL7 profiles and OCP profiles.

**Comment by shawndwells** [comment]
@redhatrises Changes look good. Last one appears to be updating ``applications/benchmark.yml`` to reflect NCP vs STIG.

**Comment by redhatrises** [comment]
> @redhatrises Changes look good. Last one appears to be updating `applications/benchmark.yml` to reflect NCP vs STIG.

done

**Comment by shawndwells** [comment]
ack pending jenkins

**Pull request merged**